src/player/PlayerQuickItem.cpp

```diff
@@ -127,7 +133,8 @@ void PlayerRenderer::render()
 
   GLint fbo = 0;
   context->functions()->glGetIntegerv(GL_FRAMEBUFFER_BINDING, &fbo);
-  bool flip = true;
+  static bool flip = true;
```

Comment:
Why is this static now?

Reply:
The parameters we pass in that struct are pointers, and the reason why mpv wants pointers is that mpv is making a copy of that struct internally.
So we want that variable to be persistent, otherwise will be lost on the copy and point to whatever is currently on the stack, flipping or not the image :)

Reply:
This is a recent change in mpv that might have been done after our update.
when i tried it with a more recent codebase, the screen would flip around like mad, which is why i made that change.

Reply:
Huh.
Could you move it to a class member instead, then?

Reply:
I guess so if you feel that's important, we don't use that anywhere else though ...

Reply:
We also don't usually use mid-function statics.